cmd/mgo/main.go

```diff
@@ -28,7 +29,14 @@ func initDB() model.Router {
 		appStorage,
 		userStorage,
 	)
-	r := ihttp.NewRouter(nil, appStorage, userStorage, tokenStorage, tokenService)
+
+	corsOptions := cors.Options{
+		AllowedHeaders: []string{"Content-Type", "X-Requested-With"},
+		AllowedOrigins: []string{"http://localhost:8080"},
+		AllowedMethods: []string{"GET", "POST", "PUT", "DELETE", "HEAD", "OPTIONS"},
+	}
+
+	r := ihttp.NewRouter(nil, appStorage, userStorage, tokenStorage, tokenService, &corsOptions)
```

Comment:
It's better to encapsulate this logic with settings

Reply:
Do you mean all params of the NewRouter method should be encapsulated in a structure named Settings?

Reply:
Or only logger and cors?